apps/console/src/features/server/pages/remote-logging-page.tsx

```diff
@@ -35,26 +36,29 @@ type RemoteLoggingPageInterface = IdentifiableComponentInterface;
 export const RemoteLoggingPage: FC<RemoteLoggingPageInterface> = (
     props: RemoteLoggingPageInterface
 ): ReactElement => {
-    
+
     const { [ "data-componentid" ]: componentId } = props;
 
     const {
         data: remoteLogPublishingConfigs,
         isLoading: isRemoteLogPublishingConfigsLoading,
-        error: remoteLogPublishingConfigRetrievalError
+        error: remoteLogPublishingConfigRetrievalError,
+        mutate: mutateRemoteLoggingRequest
     } = useRemoteLogPublishingConfigs();
 
+    const [ activeTab, setActiveTab ] = useState<TabProps["activeIndex"]>(0);
```

Comment:
Shall we use `tab-id` s instead.

```jsx
{
    "data-tabid": "audit-logs",
    menuItem: "Audit Logs",
    render: () => (
        <RemoteLoggingConfigForm
            logType={ LogType.AUDIT }
            logConfig={ remoteLogPublishingConfigs?.find(
                (config: RemoteLogPublishingConfigurationInterface) => config.logType === LogType.AUDIT
            ) }
        />    
    )
},
```

Reply:
fixed with https://github.com/wso2/identity-apps/pull/5183/commits/acfffd49ebcb00931594a3f8697d8a130e05f13e